generate-workflow.py

```diff
@@ -92,6 +92,14 @@ class BenchmarkInfo(NamedTuple):
         friendly_name='PtrDist',
         dir_name='ptrdist-1.1',
         build_cmds=textwrap.dedent(f'''\
+        ( cd yacr2 ; \\
```

Comment:
Actually maybe add a comment explaining this?

Reply:
what's your opinion on having the comment in the `build_cmds` string and therefore printed into yaml vs. only in the python?

Reply:
I don't feel strongly.  Slight preference for Python only so that if we ever want to diff the actual workflow file, we only have to review 4 copies of the code, not of the code and a long comment.  Obviously there are trade-offs, e.g., if the Python string literal is long and you want to comment on something in the middle of it.